js/pie_utils.js

```diff
@@ -69,23 +61,35 @@ function pieData(geodata) {
   var pieAll = [];
   
     for (var i=0; i<geodata.features.length; i++) {
-        pieAll.push( {
-          piechartdata: [
-            {"category": "thermoelectric", 
-                "value": geodata.features[i].properties.thermoelectric},
-            {"category": "publicsupply", 
-                "value": geodata.features[i].properties.publicsupply},
-            {"category": "irrigation", 
-                "value": geodata.features[i].properties.irrigation},
-            {"category": "industrial", 
-                "value": geodata.features[i].properties.industrial},
-            {"category": "other", 
-                "value": geodata.features[i].properties.other}],
-          piechartmeta: [
-            geodata.features[i].geometry.coordinates,
-            geodata.features[i].properties.total
-          ]
-        });
+      properties = geodata.features[i].properties;
+      var proj = projection(geodata.features[i].geometry.coordinates);
```

Comment:
I was using `projectX` and `projectY` so that any error handling for bad projections would carry through.